bin/configs/csharp-generichost-latest-nrt-useDateTimeForDate.yaml

```diff
@@ -0,0 +1,13 @@
+# for csharp generichost
+generatorName: csharp
+outputDir: samples/client/petstore/csharp/OpenAPIClient-generichost-latest-nrt-useDateTimeForDate
+inputSpec: modules/openapi-generator/src/test/resources/3_0/csharp/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml
```

Comment:
instead of using the csharp fake pestore spec (which contains many other test cases), what about using a new spec tailor made for this option to ensure it works?

Reply:
This is a huge spec so it should adequately test the change. Why would a new spec be better?

Reply:
The spec is huge and result in too many files (not needed) included in the project just to test one additional option.

The project now contains too many files (and take a long time to clone) which can be eliminated/reduced if we use a smaller spec to test the change

Reply:
I agree with @wing328
I'll make a new spec with only some date/date-time cases.
-> This is a huge spec so it should adequately test the change.

Reply:
Done.